docs/usage.md

```diff
@@ -1,117 +1,179 @@
+results         # Finished results (configurable, see below)
 .nextflow_log   # Log file from Nextflow
-.nextflow/      # Nextflow cache and history information
+# Other nextflow hidden files, eg. history of pipeline runs and old logs.
 ```
 
+### Updating the pipeline
 
-## Inputs and outputs
-
-### `--reads`
-Location of the input FastQ files:
+When you run the above command, Nextflow automatically pulls the pipeline code from GitHub and stores it as a cached version. When running the pipeline after this, it will always use the cached version if available - even if the pipeline has been updated since. To make sure that you're running the latest version of the pipeline, make sure that you regularly update the cached version of the pipeline:
 
 ```bash
- --reads 'path/to/data/sample_*_{1,2}.fastq'
+nextflow pull nf-core/methylseq
 ```
 
-**NB: Must be enclosed in quotes!**
+### Reproducibility
 
-Note that the `{1,2}` parentheses are required to specify paired end data. The file path should be in quotation marks to prevent shell glob expansion.
+It's a good idea to specify a pipeline version when running the pipeline on your data. This ensures that a specific version of the pipeline code and software are used when you run your pipeline. If you keep using the same tag, you'll be running the same version of the pipeline, even if there have been changes to the code since.
 
-If left unspecified, the pipeline will assume that the data is in a directory called `data` in the working directory (`data/*{1,2}.fastq.gz`).
+First, go to the [nf-core/methylseq releases page](https://github.com/nf-core/methylseq/releases) and find the latest version number - numeric only (eg. `1.3.1`). Then specify this when running the pipeline with `-r` (one hyphen) - eg. `-r 1.3.1`.
 
-### `--singleEnd`
-By default, the pipeline expects paired-end data. If you have single-end data, specify `--singleEnd` on the command line when you launch the pipeline. A normal glob pattern, enclosed in quotation marks, can then be used for `--reads`. For example: `--singleEnd --reads '*.fastq'`
+This version number will be logged in reports when you run the pipeline, so that you'll know what you used when you look back in the future.
 
-It is not possible to run a mixture of single-end and paired-end files in one run.
+## Main arguments
 
-### `--outdir`
-The output directory where the results will be saved.
+### `-profile`
 
-## Software
-There are different ways to provide the required software dependencies for the pipeline. The recommended method is to use either Docker or Singularity. The command line flags below are only needed if you are using the `standard` config profile (see below). It is not required with the the other profiles.
+Use this parameter to choose a configuration profile. Profiles can give configuration presets for different compute environments. Note that multiple profiles can be loaded, for example: `-profile docker` - the order of arguments is important!
```

Comment:
Hah, fail 🤦‍♂ - I guess that was meant to be `test,docker`

Order is important because later config profiles can overwrite earlier ones, yes. Will spell this out a bit more.